.github/workflows/run-tests.yml

```
@@ -10,7 +10,7 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: [7.3, 7.4, 8.0, 8.1, 8.2]
+                php: [8.0, 8.1, 8.2]
                 laravel: [7.*, 8.*, 9.*, 10.*]
```

```suggestion
                php: [8.1, 8.2]
                laravel: [10.*]
```
let's just drop support for all old Laravel versions as well (and PHP 8.0 too because Laravel 10 requires php 8.1 or later)